relay/kusama/src/lib.rs

```diff
@@ -2040,6 +2041,18 @@ sp_api::impl_runtime_apis! {
 				key_ownership_proof,
 			)
 		}
+
+			fn minimum_backing_votes() -> u32 {
+			parachains_runtime_api_impl::minimum_backing_votes::<Runtime>()
```

Comment:
Formatting seems off.

Reply:
Done in 9f5cef432e41972579d394c66cee74d8da2043e1. Not sure why this wasn't caught by rustfmt.